region/maridia/outer.json

```diff
@@ -3008,7 +3091,191 @@
           "nodeSubType": "passage",
           "nodeAddress": "0x001a45c",
           "doorEnvironments": [{"physics": "water"}],
-          "note": "It's not quite a door, but it is a morph passage transition to another room"
+          "note": "It's not quite a door, but it is a morph passage transition to another room",
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Sciser",
+                  "notable": false,
+                  "requires": [ "h_EverestMorphTunnelExpanded" ]
+                }
+              ]
+            }
+          ],
+          "gModeImmobile": {
+            "requires": [
+              "h_EverestMorphTunnelExpanded",
+              {"enemyDamage": {
+                "enemy": "Sciser",
+                "type": "contact",
+                "hits": 1
+              }}
+            ]
+          },
+          "leaveWithGMode": [
+            {
+              "leavesWithArtificialMorph": true,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Morph Through Tunnel From Top Left",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "h_EverestMorphTunnelExpanded",
```

Comment:
This doesn't seem required.  You would want the space when entering but its not needed leaving, and already in GMode.  (Leaving and not it GMode you would need the space to stand up)

Reply:
There are strats for reaching 6 in artificial morph, but they could also be used to leave in GMode, no?

Reply:
Correct. The `leaveWithGMode` section, from lines 3116-3277 are for strats getting direct G-Mode in Everest and carrying it through the tunnel. These don't actually need `h_EverestMorphTunnelExpanded`

The other strats in the links section will(?) require it, because it technically has you exit G-Mode before touching the room transition tiles (although, I don't think that actually matters, so maybe it could be removed there too?)

Reply:
If I understand, you're proposing having a separate notation like "e_EverestMorphTunnelExpanded" ("e" for edit) which behaves equivalently to the helper, just written differently to make it clear that this is only a randomizer-level toggle and not something with arbitrary requirements (such as tech, items. etc.). I think that would be fine but don't see too much difference from just using a helper, which has the advantage that it doesn't require adding complexity to the schema. Maybe for now we could use a helper, to avoid blocking adding these strats, and then potentially migrating them to a different format could be a follow-up discussion? 

I think we can consider this helper as representing the edits to these rooms specifically the way that Map Rando did it. This doesn't prevent us from potentially supporting alternative types of edits to these tunnels in the future; supporting another one would just require adding another helper for it.

I do like the idea of having randomizer-specific tweaks being something handled within the repo if possible. Requiring each randomizer to either fork the data or apply heavy patches to it would be harder to maintain. For example, if we renamed a tech or restructured nodes in a room, it would be a lot harder if you have to track down multiple repos affected and change them all.

Reply:
I agree that it would be nice to move them to their own file, so it is easier for people adopting the logic to easily see what they should/shouldn't edit. Currently it's just the first ~5 helpers. Similarly, we could pull out a separate file for leniency factors. I can make an issue for this topic.

Reply:
I think I do only want it organized differently so its ok to stay as a helper for now.  The camera fixes feel like they make a better default state to build from even though they are changes.  We do want to make it more clear what kinds of things can be readily changed.